sites.py

```diff
@@ -15,6 +15,36 @@
     from an exposé page.
 """
 sites = [
+    {
+        "name": "Wohnungsbaugenossenschaft DPF eG",
+        "url": "https://www.dpfonline.de/interessenten/immobilien/",
+        "success-str": '<div class="immo-a-info">',
+        "expose-url-pattern": r'<a href="https://www\.dpfonline\.de/(immobilien/.+?)/">',
+        "expose-details": {
+            "title": r'<h1 class="immo-caption">\s+?(.+?)\s+?</h1>',
+            "price": r'<td>Kaltmiete</td>\s+?<td>\s+?(.+?)\s+?</td>',
+            "total_rent": r'<td>Gesamtmiete</td>\s+?<td>\s+?(.+?)\s+?</td>',
+            "safety": r'<td>Genossenschaftsanteile</td>\s+?<td>\s+?(.+?)\s+?</td>',
+            "rooms": r'<td>Zimmer</td>\s+?<td>\s+?(.+?)\s+?</td>',
+            "area": r'<td>Wohnfläche</td>\s+?<td>\s+?(.+?) m<sup>2</sup>\s+?</td>',
+            "floor": r'<td>\s+?(.+?)\. Etage\s+?</td>',
+            "location": r'<td>Straße</td>\s+?<td>\s+?(.+?)\s+?</td>',
+            "quarter": r'<td>Stadtteil</td>\s+?<td>\s+?(.+?)\s+?</td>',
+        },
+    },
+    {
+        "name": "Bau- und Siedlungsgenossenschaft Postheimstätte eG",
+        "url": "https://www.postheimstätte.de/properties/",
+        "none-str": "Es tut uns leid, es wurden keine Objekte gefunden.",
+    },
+    {
+        "name": "Wohnungsbaugenossenschaft Zentrum eG",
+        "url": "https://www.wbg-zentrum.de/wohnen/wohnungsangebot-2/wohnungsangebot/",
+        "success-str": '<div class="wpb_text_column wpb_content_element ">',
+        "expose-url-pattern":   r'href="(https://www\.wbg-zentrum\.de/wp\-content/'
+                                r'uploads/.+?/.+?/.+?\.pdf)" title="" target="_blank">'
+                                r'weiter</a>',
```

Comment:
Don't try to outsmart _black_, just write like you would, and leave all formatting to it. If black thinks the string should be longer than the line length limit, then that's the right formatting.

I joined those lines into one in my version of this commit.

Reply:
If you do want multiline in cases like this, because that's easier to read for you, then use the following trick to force black's hand:

```python
{
    "expose-url-pattern": (
        r"string 1"
        r"string 2"
    ),
}